language/google/cloud/language/api_responses.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Classes representing each of the types of responses returned from
+the Natural Language API.
+"""
+
+from google.cloud.language.entity import Entity
+from google.cloud.language.sentence import Sentence
+from google.cloud.language.sentiment import Sentiment
+from google.cloud.language.syntax import Token
+
+
+class EntityResponse(object):
+    """A representation of a response sent back from the
+    ``analyzeEntites`` request to the Google Natural language API.
```

Comment:
The "no docstring in unit test" argument applies here as well: until now, we have totally avoided one-line docstring summaries that spill onto multiple lines. It seems a pity to introduce that phenomenon for this docstring, which can be easily rewritten without losing meaning.